sections/futures/TradeCrossMargin/EditCrossMarginLeverageModal.tsx

```diff
@@ -122,56 +119,8 @@ export default function EditLeverageModal({ onDismiss, editMode }: DepositMargin
 	);
 
 	const onConfirm = useCallback(async () => {
-		setError(null);
-		if (editMode === 'existing_position' && position?.position) {
```

Comment:
losing this logic means it tries to submit a transaction when in "new position" mode.